src/Microsoft.DotNet.Build.Tasks/PackageFiles/PublishProduct.targets

```diff
@@ -30,13 +30,18 @@
 
     <ItemGroup>
       <NuGetPushCommand Include="@(PackagesToPush -> '$(NuGetExePath) $(NuGetPushArgsBase) %(Identity)')" />
+
+      <IgnorableErrorMessages Include="Overwriting existing packages is forbidden according to the package retention settings for this feed.">
```

Comment:
I think here would be a good place to comment about why it's configured this way, rather than in the task file.